config.dist.yml

```diff
@@ -11,4 +11,6 @@ config:
     - 'bio'
   extractEmailDomain: true
   keepExcludedUsers: false
-  exclusions: []
+  excludeRepositories:
+    - "PrestaShop/PrestaShop-1.6"
```

Comment:
FYI there are others:

- https://github.com/PrestaShop/PrestaShop-1.5
- https://github.com/PrestaShop/PrestaShop-1.4

Reply:
Oh I see that it skips archived repositories

Reply:
Correct ;)